browser/src/layer/tile/TilesSection.ts

```diff
@@ -625,6 +649,7 @@ class TilesSection {
 			var toScaleAbs = relScale * docLayer._tileSize * 15.0 / docLayer._tileWidthTwips;
 			toScaleAbs = docLayer._tileSize * 15.0 / Math.round(15.0 * docLayer._tileSize / toScaleAbs);
```

Comment:
It would seem toScaleAbs is not used. Is this a bug?

Reply:
These lines are not from this PR, but from older zoom-animation related changes. Looks like they weren't removed after factoring some code to private utility functions (scalePosForZoom etc). I'll remove them in a follow up.